watson/cli.py

```diff
@@ -607,7 +610,14 @@ def log(watson, current, from_, to, projects, tags, year, month, week, day):
             for frame in frames
         ))
 
-    click.echo_via_pager('\n'.join(lines))
+    num_term_rows, _ = [int(i)
+                        for i in os.popen('stty size', 'r').read().split()]
+    num_rows = len(watson.frames.dump()) + (len(lines)+1) / 3 * 2 - 1
```

Comment:
This is a very hacky and un-elegant way of determining the number of rows Watson should print to the terminal. I'm going to have to find a better way to do this. Open to suggestions if anyone sees an easy way to determine the number of rows.